### DISCUSSION TITLE
Welcome to react Discussions!

### DISCUSSION THREAD

**Discussion by akaday** [akaday/react#17] [Announcements]
yarn install
yarn install v1.22.22
[1/4] Resolving packages...
success Already up-to-date.
$ node node_modules/fbjs-scripts/node/check-dev-engines.js package.json && node ./scripts/flow/createFlowConfigs.js
Done in 1.77s.
PS C:\Users\start\react>

**Comment by Anatolii-Ovcharuk** [comment]
Judging by the output, the yarn install command worked successfully, all dependencies were updated, and additional scripts also ran without errors.

**Comment by juhinagpure** [comment]
The message indicates that Your yarn install completed successfully. All dependencies are up-to-date, and setup scripts ran without issues. You're ready to start developing !
